airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/IntegrationRunner.java

```diff
@@ -145,8 +144,11 @@ public void runInternal(final ITransaction transaction, final IntegrationConfig
         final JsonNode config = parseConfig(parsed.getConfigPath());
         validateConfig(integration.spec().getConnectionSpecification(), config, "WRITE");
         final ConfiguredAirbyteCatalog catalog = parseConfig(parsed.getCatalogPath(), ConfiguredAirbyteCatalog.class);
-        final AirbyteMessageConsumer consumer = destination.getConsumer(config, catalog, outputRecordCollector);
-        AirbyteSentry.executeWithTracing("WriteDestination", () -> consumeWriteStream(consumer));
+        // use a Scanner that only processes new line characters to strictly abide with the
+        // https://jsonlines.org/ standard
```

Comment:
Why is this comment being added here?

I don't see related code to scanner being inserted at this location? (the comment is also placed right before the line 163